### PR TITLE
Feature/39 add azure ml connection

### LINKED ISSUE #39: Add AzureML to Exasol connection Example
Add a Jupyter Notebook example showcasing how to connect AzureML to an Exasol Database and access the data from ExasolDB in AzureML.
This should include all functionality and key-words.

Might be similar to [ConnectSagemakerToExasol.ipynb](https://github.com/exasol/data-science-examples/blob/main/tutorials/machine-learning/python/sagemaker/ConnectSagemakerToExasol.ipynb) and [LoadExampleDataIntoExasol.ipynb](https://github.com/exasol/data-science-examples/blob/main/tutorials/machine-learning/python/sagemaker/LoadExampleDataIntoExasol.ipynb)

### PR DESCRIPTION
**Pull request by MarleneKress79789**
Fixes #39 

Will be made pretty in #47

### CODE REVIEW

File: tutorials/machine-learning/python/AzureML/ConnectAzureMLtoExasol.ipynb [hunk 1]
```diff
@@ -0,0 +1,246 @@
+{
```
Comment (tkilias) [inline tutorials/machine-learning/python/AzureML/ConnectAzureMLtoExasol.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.        data = exasol.export_to_pandas(f"SELECT * FROM TABLE IDA.{table}")
</pre><p>This has scalability issues, if the table is large. Can we use export statements or batch the export.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-examples/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

File: tutorials/machine-learning/python/AzureML/Introduction.ipynb [hunk 2]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (tkilias) [inline tutorials/machine-learning/python/AzureML/Introduction.ipynb:1]: <p>I think, we used in the other tutorials pyexasol for uploading the test data, I think, at the moment this is the easiest way.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-examples/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: tutorials/machine-learning/python/AzureML/ConnectAzureMLtoExasol.ipynb (hunk 1 above)
Comment (MarleneKress79789) [reply tutorials/machine-learning/python/AzureML/ConnectAzureMLtoExasol.ipynb:1]: done

File: tutorials/machine-learning/python/AzureML/Introduction.ipynb (hunk 2 above)
Comment (tkilias) [reply tutorials/machine-learning/python/AzureML/Introduction.ipynb:1]: <p>ah ok, this is already your plan, if I understand this correct</p>

Comment (MarleneKress79789) [reply tutorials/machine-learning/python/AzureML/Introduction.ipynb:1]: yep :)

**Review by tkilias — APPROVED** [review]

**Pull request merged**